tests/installation/installer_desktopselection.pm

```diff
@@ -16,8 +16,11 @@ sub run() {
     assert_screen 'desktop-selection';
     my $d = get_var('DESKTOP_MINIMALX_INSTONLY') ? 'minimalx' : get_var('DESKTOP');
     if ($d ne 'kde' && $d ne 'gnome') {
-        send_key_until_needlematch 'selection_on_desktop_other', 'tab';    # Move the selection to 'Other'
-        send_key 'spc';                                                    # open 'Other' selection'
+        # up to 42.1 textmode was below 'other'
+        if (!($d eq 'textmode' && check_screen 'has-server-selection')) {
```

Comment:
I didn't want a static check as that would mean checking for versions or set variables on various media
I can reduce the timeout to 1 though as the screen doesnt change
assert_screen doesn't cover the area of the server selection